README.md

```diff
@@ -64,9 +64,12 @@ A curated list of anime streaming/downloading websites.
 * [station.f-n.me](https://station.f-n.me/)
 * [thehylia.com](https://anime.thehylia.com/)
 * [watchcartoononline.io](https://www.watchcartoononline.io/)
+* [viewster.com](https://www.viewster.com/)
```

Comment:
The content seems to limited and I see some issues with it.

![image](https://user-images.githubusercontent.com/21009455/46454791-a1182900-c7c5-11e8-9136-9fb8e935a290.png)


Can you please remove it instead?

Reply:
Content does seem to be pretty limited. I will rule this one out.